.gitignore

```diff
@@ -1,11 +1,67 @@
+# [generated] Bracketed sections updated by Yeoman generator
+# generator-canonical-webteam@2.1.2
+
+# [os] OS & editor files
+Desktop.ini
+Thumbs.db
+._*
+*.DS_Store
+*~
+\#*\#
+.AppleDouble
+.LSOverride
+.spelling
+.vscode
+
+# [cache] Cache and backup
+*.bak
+*.pyc
+*-cache/
+
+# [data] Local data
+*.sqlite*
+*.log
+logs/
+pids
+*.pid
+*.seed
+.*-metadata
+
+# [deps] Local dependencies
+.bundle/
+node_modules/
+vendor/
+bower_components/
+vendor/
+# Normally lockfiles would be committed, but we use yarn instead of NPM for locking dependencies
+package-lock.json
+
+# [build] Built files
+/build/
+_site/
+*.*.map
 build/*
-env3/*
+
+# [env] Local environment settings
+.docker-project
+.*.hash
+.envrc
+env/
+env[23]/
+
+# [sass] Files generated by Sass
+*.css
+
+# [buidl] files generated by git-repo
```

Comment:
Please remove `[buidl]` from this title. The point of the `[XXX]` tags is that they are used [by generator-canonical-webteam](https://github.com/canonical-webteam/generator-canonical-webteam/blob/master/generators/run/index.js#L86) to update blocks. But local ignores shouldn't be touched by generator, so they shouldn't have tags, and especially shouldn't have the `[build]` tag which is already a section.